saleor/graphql/utils/filters.py

```diff
@@ -60,3 +60,15 @@ def filter_range_field(qs, field, value):
         lookup = {f"{field}__lte": lte}
         qs = qs.filter(**lookup)
     return qs
+
+
+def filter_by_id(object_type):
```

Comment:
Should we use this function in all `IDs` filters?

Reply:
@stnatic Can you please open a ticket on your list for this? I can merge it as it is, but since this is rather an easy thing to reuse, we should do it as soon as we can. Please share the ticket (or add me as a watcher) when it's added.